src/main/java/com/rtosProject2/Plane.java

```diff
@@ -127,9 +127,9 @@ private void SetOccupiedFormat(Label label, boolean isCollision) {
 
                 break;
             case Y:
-                if (!halty && _atRow + 1 == 2)
+                if (!haltY && _atRow + 1 == 2)
                     System.out.println("Y is at 2");
-                if (halty){
+                if (haltY){
```

Comment:
We could probably put this if first and make the current if and elsif. Might flow a little better for other people reading. We can take care of this in part 3.